lambda/orc_ingestion.py

```diff
@@ -0,0 +1,86 @@
+import json
+import logging
+import os
+import uuid
+from datetime import datetime, timezone
+
+import boto3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+sqs_client = boto3.client("sqs")
+s3_client = boto3.client("s3")
+
+
+def lambda_handler(event, context):
+    previous_ods_code = "M85019"
```

Comment:
Could it be better so have this come through an environment variable over a hard-coded string value?